pkg/router/util/util.go

```diff
@@ -42,6 +42,7 @@ func GetIngressSpec(namespace string, trigger *fv1.HTTPTrigger) *v1.Ingress {
 	}
 
 	var ingTLS []v1.IngressTLS
+	var pathType v1.PathType = "ImplementationSpecific"
```

Comment:
We can avoid hardcoding with direct reference to the constant.
```
var pathType v1.PathType = v1.PathTypeImplementationSpecific
```

Please refer to comment https://github.com/fission/fission/pull/2164#issuecomment-901582182